openapi/shopify.admin/types.bal

```diff
@@ -955,9 +955,6 @@ public type ClientDetails record {
     string? user_agent?;
 };
 
-# An ordered list of stacked discount applications.
-public type DiscountApplications DiscountApplication[]?;
```

Comment:
@RolandHewage If we remove this type definition, it can break the existing user code right? Can't we keep it as it is?

Reply:
@Mirage20  This won't be a breaking since `DiscountApplications` was functioning as a reference **type** for `DiscountApplication` array earlier. Now we just removed the intermediate reference.

Reply:
But if a consumer uses the `DiscountApplications`, it can break due to missing type def, right?


For example,
```ballerina
DiscountApplications discountApps = order?.discount_applications;
```

Reply:
yes. missed this

We have two options to resolve this
-  Keep the type by manually adding after connector generation.
-  A major version bump.

We'll address this either one option.

Thanks @Mirage20

Reply:
This has been resolved by a major version update https://github.com/ballerina-platform/openapi-connectors/pull/702